Automattic/pocket-casts-ios
# End of Year: listened categories design

| 📘 Project: #376 |
|:---:|

Add the design for the "listening time" story.

| iPhone 8 | iPhone 14 | iPhone 14 Pro Max |
| -------- | ---------- | ------------------- |
| ![Simulator Screen Shot - iPhone 8 - 2022-10-27 at 09 17 22](https://user-images.githubusercontent.com/7040243/198282120-b16fc985-50c9-4a50-9b55-1aa45d8b841a.png) | ![Simulator Screen Shot - iPhone 14 - 2022-10-27 at 09 17 29](https://user-images.githubusercontent.com/7040243/198282165-6a71c394-d44e-4a6e-8354-024bb2069ed9.png) | ![Simulator Screen Shot - iPhone 14 Pro Max - 2022-10-27 at 09 17 37](https://user-images.githubusercontent.com/7040243/198282189-9dd7bef0-7af8-4cbf-aacf-815073e39579.png) |

## Details

* The podcasts cover displayed are determined by getting the list of top categories and reversing it, picking the top 1 podcast for each of them

## To test

I recommend using these devices to test, given they provide different screen sizes:

* iPhone 8
* iPhone 14
* iPhone 14 Pro Max

1. Enable the `endOfYear` flag in `FeatureFlag.swift`
2. Run the app and make sure you're logged in to an account with a few items on Listening History
3. When the prompt appears, tap "View My 2022"
4. Navigate to the third story
4. ✅ Check the design

## Checklist
